ROADMAP.md

```diff
@@ -15,23 +15,55 @@
 The following table includes the current roadmap for Velero. If you have any questions or would like to contribute to Velero, please attend a [community meeting](https://velero.io/community/) to discuss with our team. If you don't know where to start, we are always looking for contributors that will help us reduce technical, automation, and documentation debt.
 Please take the timelines & dates as proposals and goals. Priorities and requirements change based on community feedback, roadblocks encountered, community contributions, etc. If you depend on a specific item, we encourage you to attend community meetings to get updated status information, or help us deliver that feature by contributing to Velero.
 
-`Last Updated: May 2020`
+`Last Updated: March 2021`
 
+#### 1.7.0 Roadmap
+The release roadmap is split into Core items that are required for the release, desired items that may be removed from the
+release and opportunistic items that will be added to the release if possible.
+
+##### Core items
+
+|Issue|Description|
+|---|---|
+|[3493](https://github.com/vmware-tanzu/velero/issues/3493)|[Carvel](https://github.com/vmware-tanzu/velero/issues/3493) based installation (in addtion to the existing *velero install* CLI).|
+|[3531](https://github.com/vmware-tanzu/velero/issues/3531)|Test plan for Velero|
+|[675](https://github.com/vmware-tanzu/velero/issues/675)|Velero command to generate debugging information.  Will integrate with [Crashd - Crash Diagnostics](https://github.com/vmware-tanzu/velero/issues/675)|
+|[2066](https://github.com/vmware-tanzu/velero/issues/2066)|CSI Snapshots GA|
+|[3285](https://github.com/vmware-tanzu/velero/issues/3285)|Support Velero plugin versioning|
+|[1975](https://github.com/vmware-tanzu/velero/issues/1975)|IPV6 support|
+
+
+
+##### Desired items
+|Issue|Description|
+|---|---|
+|[3533](https://github.com/vmware-tanzu/velero/issues/3533)|Upload Progress Monitoring|
+|[2922](https://github.com/vmware-tanzu/velero/issues/2922)|Plugin timeouts|
+|[3500](https://github.com/vmware-tanzu/velero/issues/3500)|Use distroless containers as a base|
+|[3535](https://github.com/vmware-tanzu/velero/issues/3535)|Design doc for multiple cluster support|
+|[3536]|https://github.com/vmware-tanzu/velero/issues/3536|
```

Comment:
Missing parens for the link https://github.com/vmware-tanzu/velero/issues/3536

Reply:
Thanks, that should be fixed now.